exampleSite/config.yaml

```diff
@@ -19,6 +19,9 @@ permalinks:
 imaging:
   quality: 99
 
+# To disable, set to a large value
+paginate: 5
```

Comment:
Just getting a moment to review. Could you say more about this?

For example, if a user wanted to disable pagination, what value would they set this to?

Reply:
Good idea!  I updated the comment.  Let me know if that makes sense or if I should adjust.